carbon-design-system/carbon
# fix(icons): update glyph only pipeline process

Closes #9079 

Updates our icon pipeline to work with glyphs placed in `src/svg`.

#### Changelog

**New**

**Changed**

- Update `icon-build-helpers` to feed through the `glyph` size info throughout the pipeline

**Removed**

#### Testing / Reviewing

- Verify the elements preview works
- Verify glyphs are rendered as only the glyph and no extra sizes, unless those SVG assets are defined
